config/default.yml

```diff
@@ -4572,10 +4571,6 @@ Style/NumericPredicate:
                  numeric comparisons.
   StyleGuide: '#predicate-methods'
   Safe: false
-  # This will change to a new method call which isn't guaranteed to be on the
-  # object. Switching these methods has to be done with knowledge of the types
-  # of the variables which rubocop doesn't have.
```

Comment:
Shouldn't this comment be left in `Safe: false`? Because `Safe: false` includes `SafeAutoCorrect: false`.